unity-client/Assets/Scripts/MainScripts/DCL/DataStore/DataStore.cs

```diff
@@ -0,0 +1,7 @@
+﻿using Variables.RealmsInfo;
+
+public static class DataStore
+{
+    static public readonly CurrentRealmVariable playerRealm = new CurrentRealmVariable();
+    static public readonly RealmsVariable realmsInfo = new RealmsVariable();
```

Comment:
Since `CurrentRealmModel` also uses `layer` and `serverName` (`RealmsVariable` has 3 other variables that maybe we are not using today but it could come in handy to quickly know the url or users count of the current realm, the `playerRealm`), I think we could avoid having `CurrentRealmVariable` and use `RealmsVariable` (that I'd call `RealmVariable`) for both of these variables, transforming `realmsInfo` into an array of `RealmVariable`.

What do you think?

Reply:
making playerRealm a `RealmsVariable` will make it trigger Change event if a new user gets into that realm (that would me annoying) and I think most of the time we will just need an event for when player changes realm and the name of the realm...
If we need more info about the current realm then we have several ways to do that like creating a property that iterates through `realmsInfo` or creating a new variable for that